scripts/stackstorm.js

```diff
@@ -238,6 +239,9 @@ module.exports = function(robot) {
       .catch(function (err) {
         var error_msg = 'Failed to retrieve commands from "%s": %s';
```

Comment:
Let's also change this to log the full traceback. See https://github.com/hubotio/hubot/blob/master/src/robot.js#L461 for an example.